packages/xo-server/src/xapi/mixins/patching.js

```diff
@@ -141,19 +164,8 @@ export default {
 
   // LIST ----------------------------------------------------------------------
 
-  // list all yum updates available for a XCP-ng host
-  // (hostObject) → { uuid: patchObject }
-  async _listXcpUpdates(host) {
-    return JSON.parse(
-      await this.call(
-        'host.call_plugin',
-        host.$ref,
-        'updater.py',
-        'check_update',
-        {}
-      )
-    )
-  },
+  _listXcpUpdates,
```

Comment:
Is this one necessary, now?

Reply:
found that people might want to add a button to force the cache